packages/disploy/src/structs/ModalActionRowComponent.ts

```diff
@@ -0,0 +1,45 @@
+import {ActionRowComponent} from "./ActionRowComponent";
+import type {App} from "../client";
+import {ComponentType, TextInputStyle} from "discord-api-types/v10";
+
+export class ModalActionRowComponent extends ActionRowComponent {
```

Comment:
I think this should be called a `TextInputComponent`. While *it is* a modal action row component, there may be others in the future.

Reply:
What about making TextInputComponent and extending ModalActionRowComponent in it?